### PR TITLE
 tidb-in-kubernetes: make deploy doc clearer on GCP

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
This PR updates deploy doc on GCP to make it clearer.

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1903

### Which version does your change affect? 
dev, v3.0

### CODE REVIEW

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 1]
```diff
@@ -324,20 +351,83 @@ GCP allows attaching a local SSD to any instance type that is `n1-standard-1` or
...
+
+```
+override_values_file = "./test-cluster.yaml"
+```
 
-For example, the default cluster uses `default.yaml` in the `gcp/tidb-cluster` module as the overriding values file, and the ConfigMap rollout feature is enabled in this file.
+The default cluster uses `values/default.yaml` in the `deploy/modules/gcp/tidb-cluster` module as the overriding values file.
 
-In GKE, some values are not customizable in `values.yaml`, such as the cluster version, replicas, node selectors, and taints. These variables are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. To customize these variables, you can edit the variables of each `tidb-cluster` module in the `tidbclusters.tf` file directly.
+In GKE, some values are not customizable in `values.yaml`, such as the cluster version, replicas, `NodeSelectors` and `Tolerations`. `NodeSelector` and `Tolerations` are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. Cluster version and replicas can be modified in each `./tidb-cluster` module in the `clusters.tf` file directly.
```
Comment (DanielZhangQD) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]: `./tidb-cluster` -> `tidb-cluster`

**Comment by anotherrachel** [comment]
@DanielZhangQD all comments addressed. PTAL again, thanks~

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 2]
```diff
@@ -324,20 +350,83 @@ GCP allows attaching a local SSD to any instance type that is `n1-standard-1` or
...
+
+```
+override_values_file = "./test-cluster.yaml"
+```
 
-For example, the default cluster uses `default.yaml` in the `gcp/tidb-cluster` module as the overriding values file, and the ConfigMap rollout feature is enabled in this file.
+The default cluster uses `values/default.yaml` in the `deploy/modules/gcp/tidb-cluster` module as the overriding values file.
 
-In GKE, some values are not customizable in `values.yaml`, such as the cluster version, replicas, node selectors, and taints. These variables are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. To customize these variables, you can edit the variables of each `tidb-cluster` module in the `tidbclusters.tf` file directly.
+In GKE, some configuration items are not customizable in `values.yaml`, such as the cluster version, replicas, `NodeSelectors` and `Tolerations`. `NodeSelector` and `Tolerations` are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. Cluster version and replicas can be modified in each `tidb-cluster` module in the `clusters.tf` file directly.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
In GKE, some configuration items are not customizable in `values.yaml`, such as the cluster version, replicas, `NodeSelector` and `Tolerations`. `NodeSelector` and `Tolerations` are controlled by Terraform to ensure consistency between the infrastructure and TiDB clusters. Cluster version and replicas can be modified in each `tidb-cluster` module in the `clusters.tf` file directly.
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 3]
```diff
@@ -324,20 +350,83 @@ GCP allows attaching a local SSD to any instance type that is `n1-standard-1` or
...
+
+> **Note:**
+>
+>It's not recommended to include the following configurations (default configurations of `tidb-cluster` module) in the customized `values.yaml`:
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
> It's not recommended to include the following configurations (default configurations of `tidb-cluster` module) in the customized `values.yaml`:
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 4]
```diff
@@ -324,20 +350,83 @@ GCP allows attaching a local SSD to any instance type that is `n1-standard-1` or
...
+>     config:
+>       GF_AUTH_ANONYMOUS_ENABLED: "true"
+>     service:
+>       type: LoadBalancer
+> ```
 
 ### Customize TiDB Operator
 
-You can customize the TiDB operator by specifying a Helm values file through the `override_values` variable. This variable can be passed into a `tidb-cluster` module.
+You can customize the TiDB operator by specifying overriding values through the `operator_helm_values` variable or specifying an overriding values file through the `operator_helm_values_file` variable. If both variables are configured, then `operator_helm_values` will be enabled and the value of which will be passed into the `tidb-cluster` module.
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
You can customize TiDB Operator by specifying overriding values through the `operator_helm_values` variable or specifying an overriding values file through the `operator_helm_values_file` variable. If both variables are configured, then `operator_helm_values` will be enabled and its value will be passed into the `tidb-cluster` module.
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 5]
```diff
@@ -447,12 +530,11 @@ The best practices for managing multiple Kubernetes clusters are:
 - Creating a new directory for each of your Kubernetes clusters, and
 - Combining the above modules according to your needs via Terraform scripts.
 
-These are so that the Terraform states among clusters do not interfere with each other, and it is convenient to expand. Here's an example:
+In this case, the Terraform states among clusters do not interfere with each other, and it is convenient to expand. Here's an example (assume you are in the project root):
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
In this case, the Terraform states among clusters do not interfere with each other, and it is convenient to expand. Here's an example (assume you are in the project root directory):
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 6]
```diff
@@ -447,12 +530,11 @@ The best practices for managing multiple Kubernetes clusters are:
...
 
 {{< copyable "shell-regular" >}}
 
 ```shell
-# assume we are in the project root
 mkdir -p deploy/gcp-staging
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]:
```suggestion
mkdir -p deploy/gcp-staging &&
```

File: dev/tidb-in-kubernetes/deploy/gcp-gke.md [hunk 7]
```diff
@@ -324,20 +344,83 @@ GCP allows attaching a local SSD to any instance type that is `n1-standard-1` or
...
+>   separateSlowLog: true
+> monitor:
+>   storageClassName: pd-ssd
+>   persistent: true
+>   grafana:
+>     config:
+>       GF_AUTH_ANONYMOUS_ENABLED: "true"
+>     service:
+>       type: LoadBalancer
+> ```
```
Comment (yikeke) [inline dev/tidb-in-kubernetes/deploy/gcp-gke.md]: remove `>`

**Comment by anotherrachel** [comment]
@yikeke All comments addressed, PTAL again~

**Pull request merged**